src/udev/udev-config.h

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+#pragma once
+
+#include <stdbool.h>
+
+extern bool arg_daemonize;
+
+typedef struct Manager Manager;
+
+int manager_load(Manager *manager, int argc, char *argv[]);
+void manager_set_default_children_max(Manager *manager);
```

Comment:
Is this needed? Can't we make it static now?

Reply:
It is also used by `on_ctrl_msg()` in udev-manager.c .